src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/RestoreOperation/RestoreUtil.cs

```diff
@@ -0,0 +1,738 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Data.Tools.DataSets;
+using Microsoft.SqlServer.Management.Sdk.Sfc;
+using Microsoft.SqlServer.Management.Smo;
+
+namespace Microsoft.SqlTools.ServiceLayer.DisasterRecovery.RestoreOperation
+{
+    public class RestoreUtil
+    {
+        public RestoreUtil(Server server)
+        {
+            this.server = server;
+            this.excludedDB = new List<string> { "master", "tempdb" };
```

Comment:
this code is copied from ssms so I'm going to change it now unless we know there's a bug in ssms

---
In reply to: [126552863](https://github.com/Microsoft/sqltoolsservice/pull/403#discussion_r126552863) [](ancestors = 126552863)